lib/client.js

```diff
@@ -115,6 +117,10 @@ class Client extends DispatcherBase {
       throw new InvalidArgumentError('unsupported keepAlive, use pipelining=0 instead')
     }
 
+    if (reset != null && typeof reset !== 'boolean') {
+      throw new InvalidArgumentError('invalid reset')
```

Comment:
in order to have a consistent error message, this should be the same as the previous error message `reset must be a boolean`